### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by rhatdan**
crun should be available in f35.

[ NO NEW TESTS NEEDED]

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL
@edsantiago @cevich @giuseppe PTAL

File: contrib/cirrus/setup_environment.sh
Comment (edsantiago) [inline contrib/cirrus/setup_environment.sh:124]: So, is the problem that this version has been wiped from mirrors? If so, does this actually do what you think it does? Since the CI VMs are (presumably) static, won't this just leave them with a very-old (pre-1.3.2) version of crun?

Let's see what CI does, but my gut is telling me that we should maybe, instead of deleting this block, `s/1.3-2/1.4-1/` at least until @cevich returns from PTO

Comment (edsantiago) [reply contrib/cirrus/setup_environment.sh:124]: Followup: yeah, [package_versions](https://cirrus-ci.com/task/6339561314844672?logs=package_versions#L9) is showing `crun-1.3.1`, which is presumably the problem that the lines above were trying to fix. I think that, until the CI VMs get rebuilt, the solution needs to be `s/1.3-2/1.4-1/`

Comment (rhatdan) [reply contrib/cirrus/setup_environment.sh:124]: Can we just do a yum update crun?

Comment (edsantiago) [reply contrib/cirrus/setup_environment.sh:124]: Yes, I think that would be even better. Good thinking, and I'm sorry for not stepping back to suggest that. Juggling too many things this morning!

**Comment by mheon** [comment]
LGTM whenever tests go green

**Review by giuseppe — APPROVED** [review]
/lgtm
/hold

**Review by vrothberg — COMMENTED** [review]
LGTM

**Pull request merged**
